openstack/cinder/templates/etc/_ratelimit.yaml.tpl

```diff
@@ -0,0 +1,98 @@
+# List of whitelisted scopes keys (domainName/projectName).
+whitelist:
+  - default/service
+  - default/admin
+  - default/l3admin
+
+# Override default ratelimit response.
+ratelimit_response:
+  status: 498 Rate Limited
+  status_code: 498
+  json_body: { "message": "Rate limit exceeded" }
+
+# Override default blacklist response.
+blacklist_response:
+  status: 497 Blacklisted
+  status_code: 497
+  json_body: { "message": "You have been blacklisted. Please contact administrator." }
+
+# Group multiple CADF actions to one rate limit action.
+groups:
+  write:
+    - update
+    - delete
```

Comment:
Where does create come in? Do we allow it unlimited or is it an `update`?

Reply:
@auhlig, do we need to explicitly specify all methods here? - in watcher I can see also: 
update/os-complete
update/os-begin_detaching
update/os-reset_status
update/os-force_delete
and so on.

Reply:
Yes. 
Rate limits must be set explicitly to avoid potential confusion why/when/if limits are implicitly applied.
But I get the point of making this more flexible

Reply:
yes, wildcards would be helpful :)